runtime/doc/lsp.txt

```diff
@@ -720,6 +720,18 @@ foldexpr({lnum})                                          *vim.lsp.foldexpr()*
     Provides an interface between the built-in client and a `foldexpr`
     function.
 
+    To use, check for the "textDocument/foldingRange" capability in an
+    |LspAttach| autocommand. Example: >lua
+        vim.api.nvim_create_autocommand('LspAttach', {
+          callback = function(args)
+            local client = vim.lsp.get_client_by_id(args.data.client_id)
+            if client.supports_method('textDocument/foldingRange') then
```

Comment:
```suggestion
            if client:supports_method('textDocument/foldingRange') then
```

Since the non-method function was deprecated.

Reply:
Addressed both in c0898f8. Thanks!